docs/useLoadDataState.md

```diff
@@ -0,0 +1,64 @@
+# OneBlink Apps - ReactJS | Usage
+
+## `useLoadDataState()`
+
+This function is a react hook for managing the state involved with loading data.
+
+## Parameters
+
+The hook takes a single function as an argument:
+
+| Property | Type       | Description                                                                     |
+| -------- | ---------- | ------------------------------------------------------------------------------- |
+| `onLoad` | `function` | The function that fetches your data. Should be a Promise that returns your data |
+
+## Return
+
+The return type of `useLoadDataState()` is an object with the following properties:
+
+| Property     | Type            | Description                                                                         |
+| ------------ | --------------- | ----------------------------------------------------------------------------------- |
+| `state`      | `LoadDataState` | The current state.                                                                  |
+| `handleLoad` | `function`      | A function that runs the `onLoad` function again. This can be used to refresh data. |
+
+`useLoadDataState()` can be used like:
+
+```js
+import { useLoadDataState } from '@oneblink/apps-react'
+import fetchData from 'fetchData' // Must be a function which returns a promise of some data
```

Comment:
Can we change this to an example of a function that returns a promise with some state in it. Using `fetch` would be a good example.

Reply:
yep

Reply:
```suggestion
const fetchData = async () => {
  const response = await fetch('https://example.com/data')
  return await response.json()
}
```